frontend/apps/filemanager/filemanager.lua

```diff
@@ -297,8 +298,42 @@ function FileManager:init()
                 }
             },
             -- a little hack to get visual functionality grouping
-            {},
+            {
+            },
         }
+
+        if lfs.attributes(file, "mode") == "file" and string.lower(FrontUtil.getFileNameSuffix(file)) == "sh" then
+            -- NOTE: We populate the empty separator, in order not to mess with the button reordering code in CoverMenu
```

Comment:
Nice trick :)

Reply:
I, err, may have lost a few hairs before realizing that I was mangling the nesting in my first attempts >_<".